temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid using `pdb` (Python Debugger) in production code. Debugging statements should be removed before committing.
```
import pdb
```